config/sitemap.rb

```diff
@@ -0,0 +1,36 @@
+#
```

Comment:
Oh I gotcha... hm. I would ask then if we can configure it  like this?

https://github.com/sul-dlss/earthworks/pull/207